README.md

```diff
@@ -94,6 +94,12 @@ making the combined instantiation/stub declaration very terse:
 double(:foo => 'bar')
 ```
 
+So, the above examples can be written simply as:
+
+```ruby
+allow(book).to receive(:title).and_return(book)
+```
+
```

Comment:
We're not alluding to this as a conclusion, as we state it up front with `allow(book).to receive(:title).and_return("The RSpec Book")`